eventnotifier/slack/slack_test.go

```diff
@@ -1019,45 +1019,521 @@ var _ = Describe("Slack Notifier", func() {
+
+						expectedSlackUser := &slack.User{
+							ID:   uuid.New().String(),
+							Name: expectedUserInfo.Username,
+						}
+						slackNotifMock.EXPECT().GetUserByEmail(expectedUserInfo.Username).Return(expectedSlackUser, nil).Once()
+
+						By("sending the message to the user")
+						slackNotifMock.EXPECT().PostMessage(
+							expectedSlackUser.ID,
+							mock.AnythingOfType("slack.MsgOption"),
+							mock.AnythingOfType("slack.MsgOption"),
+							mock.AnythingOfType("slack.MsgOption"),
+							mock.AnythingOfType("slack.MsgOption"),
+							mock.AnythingOfType("slack.MsgOption"),
+						).Return("", "", nil).Once()
+
+						//Act
+						err := notifier.Notify(obj,
+							corev1.Event{},
+							notifType,
+						)
+
+						Expect(err).ShouldNot(HaveOccurred())
+
+					},
+						Entry("a success notification with a disruption", &v1beta1.Disruption{
+							TypeMeta: metav1.TypeMeta{
+								Kind: v1beta1.DisruptionKind,
+							},
+						}, types.NotificationSuccess),
+						Entry("a completion notification with a disruption", &v1beta1.Disruption{
+							TypeMeta: metav1.TypeMeta{
+								Kind: v1beta1.DisruptionKind,
+							},
+						}, types.NotificationCompletion),
+						Entry("a unknown notification with a disruption", &v1beta1.Disruption{
+							TypeMeta: metav1.TypeMeta{
+								Kind: v1beta1.DisruptionKind,
+							},
+						}, types.NotificationUnknown),
+						Entry("a warning notification with a disruption", &v1beta1.Disruption{
+							TypeMeta: metav1.TypeMeta{
+								Kind: v1beta1.DisruptionKind,
+							},
+						}, types.NotificationWarning),
+						Entry("an error notification with a disruption", &v1beta1.Disruption{
+							TypeMeta: metav1.TypeMeta{
+								Kind: v1beta1.DisruptionKind,
+							},
+						}, types.NotificationError),
+						Entry("a success notification with a disruptionCron", &v1beta1.DisruptionCron{
+							TypeMeta: metav1.TypeMeta{
+								Kind: v1beta1.DisruptionCronKind,
+							},
+						}, types.NotificationInfo),
+						Entry("a completion notification with a disruptionCron", &v1beta1.DisruptionCron{
+							TypeMeta: metav1.TypeMeta{
+								Kind: v1beta1.DisruptionCronKind,
+							},
+						}, types.NotificationCompletion),
+						Entry("a unknown notification with a disruptionCron", &v1beta1.DisruptionCron{
+							TypeMeta: metav1.TypeMeta{
+								Kind: v1beta1.DisruptionCronKind,
+							},
+						}, types.NotificationUnknown),
+						Entry("a warning notification with a disruptionCron", &v1beta1.DisruptionCron{
+							TypeMeta: metav1.TypeMeta{
+								Kind: v1beta1.DisruptionCronKind,
+							},
+						}, types.NotificationWarning),
+						Entry("an error notification with a disruptionCron", &v1beta1.DisruptionCron{
+							TypeMeta: metav1.TypeMeta{
+								Kind: v1beta1.DisruptionCronKind,
+							},
+						}, types.NotificationError),
+					)
+
+				})
+
+				Context("slackUserEmail differs from the userInfoUsername", func() {
```

Comment:
How is this different from `the UserInfo username and SlackUserEmail are configured but differ in value`?